twingate/internal/provider/resource/helper.go

```diff
@@ -24,3 +24,37 @@ func convertIDs(data interface{}) []string {
 		},
 	)
 }
+
+// setIntersection - for given two sets A and B,
+// A ∩ B (read as A intersection B) is the set of common elements that belong to set A and B.
+// If A = {1, 2, 3, 4} and B = {3, 4, 5, 7}, then the intersection of A and B is given by A ∩ B = {3, 4}.
+func setIntersection(a, b []string) []string {
```

Comment:
need to move its tests from `resource_test.go` to a new `helpers_test.go`

Reply:
moved test to helpers_test